<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessManual</class>
 <widget class="QWidget" name="ChessManual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>738</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1034</width>
    <height>738</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1034</width>
    <height>738</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ChessManual{background-image: url(:/pic/chessBoard.png);
background-repeat:no-repeat;}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>720</y>
     <width>1031</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="statusLayout"/>
  </widget>
  <widget class="QGraphicsView" name="posTest">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>180</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* background-image: url(:/pic/blackPiecePredrop.png); */
background-color:rgba(0,0,0,0);
border:none;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>610</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
background-image: url(:/pic/buttonBackground.png);
color:rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string>返回主页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>540</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
background-image: url(:/pic/buttonBackground.png);
color:rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="chessRecord">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>80</y>
     <width>256</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
